galpy/potential/potential_c_ext/galpy_potentials.h

```diff
@@ -8,6 +8,7 @@
 extern "C" {
 #endif
 #include <interp_2d.h>
+#include <sf_math.h>
```

Comment:
Don't think this is necessary here?

Reply:
I started by adding <sf_math.h> wherever interp_2d appeared to ensure that the addition worked.

Reply:
re-adding it to test why the build errored.